tests/unit/src/test/scala/tests/DidFocusLspSuite.scala

```diff
@@ -205,10 +208,6 @@ class DidFocusWhileCompilingLspSuite
       // And make sure didFocus during the compilation causes compilation against project b.
       didCompile <- server.didFocus("b/src/main/scala/b/B.scala")
       _ <- didSaveA
-      _ = assert(
```

Comment:
Should we remove the assert?

Reply:
Yeah, we have to, it wasn't pass otherwise. Because we never focus away from `B.scala`, it will get compiled anyway and focusing again on that file won't cause it to recompile. It doesn't matter the effect is the same.